src/modules/claims/claims.service.ts

```diff
@@ -1354,22 +1358,40 @@ export class ClaimsService {
     );
   }
 
-  /**
+    /**
+   * Verify the issuer and chain of trust for a Verifiable Credential (VC)
+   *
+   * @param {VerifiableCredential<RoleCredentialSubject} vc to be verified
+   */
+     async verifyVc(vc: VerifiableCredential<RoleCredentialSubject>) {
+      const issuerDID = this._signerService.did;
+      const role = vc.credentialSubject.role.namespace;
+      if (
+        !(
+          (await this._vcIssuerVerifier.verifyIssuerAuthority(role, issuerDID)) ||
```

Comment:
Recently we changed this API. Now it is used like this
```suggestion
          (await this._vcIssuerVerifier.verifyIssuance(issuerDID, role)) ||
          (await this._vcIssuerVerifier.verifyIssuer(issuerDID, role))
```

Reply:
@whitneypurdum you might need to update the `@energyweb/vc-verification` for this

Reply:
Looks like verifyChainOfTrustByRoleDefinition is not used either? @JGiter

Reply:
yes, two lines which I suggested are replacement for whole if condition